verification/curator-service/ui/src/components/App.tsx

```diff
@@ -79,6 +80,11 @@ class App extends React.Component<{}, User> {
                                 <SourceTable />
                             </Route>
                         )}
+                        {this.hasAnyRole(['reader']) && (
```

Comment:
client side role checks won't give us much as there's no "reader" role check when trying to access the charts on the mongo side. If somebody has the URL they can view it, so I'm not sure we need this check here it just seems to add toil as we'll have to ask admins to set the reader role on people.
Consider removing?

Reply:
Sounds good, done.